common/app/model/content.scala

```diff
@@ -567,6 +567,7 @@ object Audio {
   }
 
   def acastUrl(player: AudioPlayer, url: String, isAdFree: Boolean): String = if (player.audio.tags.isPodcast && !isAdFree) "https://flex.acast.com/" + url.replace("https://", "") else url
+  def acastUrl(url: String): String = "https://flex.acast.com/" + url.replace("https://", "")
```

Comment:
We probably still want the `isAdFree` test in there, for those lucky guys who are in both cohorts at the same time 👼